### PR TITLE
DC-569 and moved prereqs to reusables

### PR DESCRIPTION
**Pull request by eltesoro-koji**
DC-569 Remove outdated content (Later fixed the tutorial link by added # to the end.)
Deleted the old startGuide*.adoc files.
Also standardized and moved prerequisites to reusables (_includes/prereqs.adoc).

### CODE REVIEW

File: src/asciidoc/docs/_includes/redirects.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+= Redirects
```
Comment (rasienko) [inline src/asciidoc/docs/_includes/redirects.adoc:1]: It looks like we’re not using this include page anywhere yet. We could think about applying this note to some of the pages in the lower-priority updates bucket. At our meeting today, let’s go through the roadmap and discuss whether we want to do it.

File: src/asciidoc/docs/_includes/prereqs.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+// end::backend[]
+
+// tag::svcmap[]
+* Basic understanding of the Koji service map.
+For an overview, see <<add-service#>>.
+// end::svcmap[]
+
+// tag::kojidb[]
+* Basic understanding of the Koji database.
+See the <<withkoji-database-package#>> package reference.
```
Comment (rasienko) [inline src/asciidoc/docs/_includes/prereqs.adoc:65]:
```suggestion
See <<koji-database#>>.
```
Updating to guide instead of the reference for the deprecated package.

File: src/asciidoc/docs/_includes/redirects.adoc (hunk 1 above)
Comment (eltesoro-koji) [reply src/asciidoc/docs/_includes/redirects.adoc:1]: Okies!

File: src/asciidoc/docs/_includes/starterExamples.adoc [hunk 3]
```diff
@@ -3,9 +3,6 @@
 // tag::all[]
 [TIP]
 ====
```
[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/docs/_includes/starterExamples.adoc:5]: Removed references to starterExamples and deleted the file.

File: src/asciidoc/docs/blueprints/customVCC.adoc [hunk 4]
```diff
@@ -10,13 +11,10 @@ image::cat-selector.png[Make a custom VCC on Koji,role="img-overview"]
...
 === Prerequisites
 
-* Familiarity with web development.
-React and ES6 basics are a plus.
-* Understanding of third-party API use is a plus.
-* Familiarity with the Koji remix process.
-For an overview, see the <<start-guide-1#, starter guide>>.
-* Familiarity with Koji custom VCCs is a bonus.
-For an overview, see <<build-custom-vcc#>>.
+include::{includespath}/prereqs.adoc[tag=webdev]
```
[the remark this answers is not in the document]
Comment (eltesoro-koji) [reply src/asciidoc/docs/blueprints/customVCC.adoc:14]: Combined prereq includes into one line. Will be in next check-in.

File: src/asciidoc/docs/_includes/prereqs.adoc (hunk 2 above)
Comment (eltesoro-koji) [reply src/asciidoc/docs/_includes/prereqs.adoc:65]: Fixed. Will be in next check-in.

File: src/asciidoc/docs/blueprints/magazineCover.adoc [hunk 5]
```diff
@@ -10,8 +10,7 @@ image::magazine-cover.png[Make a magazine cover template on Koji,role="img-overv
 
 === Prerequisites
 
-include::{includespath}/prereqs.adoc[tag=webdev]
-include::{includespath}/prereqs.adoc[tag=remixproc]
+include::{includespath}/prereqs.adoc[tag=webdev;remixproc]
```
Comment (rasienko) [inline src/asciidoc/docs/blueprints/magazineCover.adoc:13]:
```suggestion
include::{includespath}/prereqs.adoc[tags=webdev;remixproc]
```
You have to use the plural for multiple tags to work. Please update on the rest of the pages, too. Be sure to run/preview the docs locally to help catch these syntax oddities before reviews.

File: src/asciidoc/tutorials/getting-started/quickStart.adoc [hunk 6]
```diff
@@ -9,7 +9,7 @@ Along the way, you'll learn some general web development topics along with the f
 NOTE: This tutorial provides a step-by-step introduction for developers who are new to Koji, with or without web development experience.
 
 ////
-If you prefer to jump straight into code, you can head to the <<start-guide-1#,advanced quick start guide>>.
+If you prefer to jump straight into code, go through <<quick-start#, the Koji quick-start tutorial>>.
```
Comment (rasienko) [inline src/asciidoc/tutorials/getting-started/quickStart.adoc:12]: Actually, let's revert this change. If you want, you can update the link to the slug you're going to use for the Advanced quick start guide (your next project). You can uncomment this part of the note as part of that ticket.

**Review by rasienko — CHANGES_REQUESTED** [review]
Couple of small things and then this will be ready for merge. Thanks!

File: src/asciidoc/docs/blueprints/magazineCover.adoc (hunk 5 above)
Comment (eltesoro-koji) [reply src/asciidoc/docs/blueprints/magazineCover.adoc:13]: Fixed. Sorry about that.

File: src/asciidoc/tutorials/getting-started/quickStart.adoc (hunk 6 above)
Comment (eltesoro-koji) [reply src/asciidoc/tutorials/getting-started/quickStart.adoc:12]: Since it's commented out, I restored the text but I prepended "adv-" to the stub (adv-quick-start) and that's what I'll use for the new page. Thanks!

**Review by rasienko — APPROVED** [review]
LGTM! Approved!
